plone/dexterity/content.py

```diff
@@ -634,20 +635,20 @@ class Item(PasteBehaviourMixin, BrowserDefaultMixin, DexterityContent):
         'action': 'view',
     },) + CMFCatalogAware.manage_options + SimpleItem.manage_options
 
-    # Be explicit about which __getattr__ to use
-    __getattr__ = DexterityContent.__getattr__
-
 
 @implementer(IDexterityContainer)
 class Container(
-        PasteBehaviourMixin, DAVCollectionMixin, BrowserDefaultMixin,
-        CMFCatalogAware, CMFOrderedBTreeFolderBase, DexterityContent):
+    DexterityContent,
+    PasteBehaviourMixin,
+    DAVCollectionMixin,
+    BrowserDefaultMixin,
+    CMFCatalogAware,
+    CMFOrderedBTreeFolderBase,
+    DexterityContent
```

Comment:
``DexterityContent`` is duplicate (first and last superclass) - is this by intention or just a copy/paste error?